wiki/about/modders.md

```diff
@@ -69,6 +69,14 @@ One of many with a very questionable sleep schedule. Creator of MorePrecisePlaye
 
 [GitHub](https://github.com/erisapps) | [Donate](https://ko-fi.com/erisapps)
 
+## Goobie
+Maintainer of the mods that help you play with friends the right way. Spends way too much time coding, and considers herself best girl.
```

Comment:
Im gonna need some documentation for best girl

Reply:
![image](https://user-images.githubusercontent.com/47616186/100949981-585d7700-34d9-11eb-8dd0-ba7af1e007e3.png)
![image](https://user-images.githubusercontent.com/47616186/100949995-63180c00-34d9-11eb-86e7-d60ddb38eeb3.png)
theres ur best girl docs smh, simp army comes in clutch